### PR TITLE
run from source with python3 instead of ppa

### PR DESCRIPTION
**Pull request by aptalca**
Once merged, the Jenkins trigger needs to be updated

### CODE REVIEW

File: jenkins-vars.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # jenkins variables
 project_name: docker-sabnzbd
 external_type: na
```
Comment (nemchik) [inline jenkins-vars.yml:5]: Can this be GitHub now? Might need other changes.

Comment (aptalca) [reply jenkins-vars.yml:5]: No, this is the pre-release one. We want it to build both pre-release and stable releases

File: Dockerfile [hunk 2]
```diff
@@ -24,32 +24,39 @@ RUN \
...
 	python3-pip \
-	${SABNZBD} \
+	python3-six \
 	unrar \
 	unzip && \
- pip3 install --no-cache-dir \
-	apprise \
-	chardet \
-	pynzb \
-	requests \
```
Comment (Safihre) [inline Dockerfile:46]: The `requests` and `apprise` packages are not listed in our `requirements.txt`, so probably good to still install them seperatlty?

File: Dockerfile [hunk 3]
```diff
@@ -24,32 +24,39 @@ RUN \
...
+	libffi-dev \
+	libssl-dev \
 	p7zip-full \
 	par2-tbb \
 	python3 \
 	python3-pip \
-	${SABNZBD} \
+	python3-six \
 	unrar \
 	unzip && \
```
Comment (Safihre) [inline Dockerfile:38]: `unzip` is no longer needed, we always use `7zip`.

Comment (aptalca) [reply Dockerfile]: thanks, removed unzip

File: Dockerfile (hunk 2 above)
Comment (aptalca) [reply Dockerfile:46]: I added them back in. I still need to make sure I don't accidentally remove some python dependency packages along with python3-pip at the end of the build

Comment (aptalca) [reply Dockerfile:46]: added cryptography and distutils as ubuntu packages so they don't get attempted to be removed along with pip

**Pull request merged**